MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,750 @@
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        ColumnCount="2"
+        >
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.ColumnCount = 2;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Setting the `ColumnCount` property to SfDataForm does not arrange the data field in a group according to the column count. To set the column count for data fields in the data form group, refer to [loading different layout for data form group]
+
+When the label position is Left, the grid layout is shown as follows:
+
+When the label position is Top, the grid layout is shown as follows:
+
+## Label visibility
+
+The label can be hided by defining the [DataFormDisplayOptions] attribute or by handling `GenerateDataFormItem` event. In this case, only the editor will be loaded.
+
+### Using attributes
+
+{% tabs %}
+{% highlight c# %}
+private double? percentage;
+
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Using event
+
+{% tabs %}
+{% highlight c# %}
+
+ this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.PlaceholderText = "Enter your name";
+            e.DataFormItem.ShowLabel = false;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Label position
+
+Label can be positioned either top or left side of the editor. By using the [DataFormDefaultLayoutSettings.LabelPosition] property, you can layout the label associated with editor.
+
+By default, the label will be positioned at left side of the editor.
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm">
+        <DataForm:SfDataForm.DefaultLayoutSettings>
+            <DataForm:DataFormDefaultLayoutSettings
+                LabelPosition="Left"/>
+        </DataForm:SfDataForm.DefaultLayoutSettings>
+    </DataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.dataForm.DefaultLayoutSettings.LabelPosition = DataFormLabelPosition.Top;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Changing label position of the DataFormItem
+
+The label position can be changed by using the [DataFormDefaultLayoutSettings.LabelPosition] property in `DataFormItem`, and it will be handled in the `GenerateDataFormItem` event.
```

Comment:
The label position of particular editor can be changed by using the [DataFormDefaultLayoutSettings.LabelPosition] property in `DataFormItem`,

Reply:
changed content